test/fail_compilation/verifyhookexist.d

```diff
@@ -0,0 +1,47 @@
+/*
+REQUIRED_ARGS: -Ifail_compilation/extra-files/minimal/ -checkaction=context
+PERMUTE_ARGS:
+*/
+
+/************************************************************/
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/verifyhookexist.d(102): Error: `object.__ArrayCast` not found. The current runtime does not support casting array of structs, or the runtime is corrupt.
+fail_compilation/verifyhookexist.d(200): Error: `object.__equals` not found. The current runtime does not support equal checks on arrays, or the runtime is corrupt.
+fail_compilation/verifyhookexist.d(201): Error: `object.__cmp` not found. The current runtime does not support comparing arrays, or the runtime is corrupt.
+fail_compilation/verifyhookexist.d(205): Error: `object._d_assert_fail` not found. The current runtime does not support generating assert messages, or the runtime is corrupt.
+fail_compilation/verifyhookexist.d(208): Error: `object.__switch` not found. The current runtime does not support switch cases on strings, or the runtime is corrupt.
+fail_compilation/verifyhookexist.d(213): Error: `object.__switch_error` not found. The current runtime does not support generating assert messages, or the runtime is corrupt.
+---
+*/
+
+#line 100
```

Comment:
Please avoid using these `#line` tokens. There's a command to automatically update the error messages: https://github.com/dlang/dmd/blob/master/test/README.md#automatically-update-the-test_output-segments